src/sentry/search/eap/utils.py

```diff
@@ -28,3 +37,29 @@ def add_start_end_conditions(
     in_msg.meta.end_timestamp.CopyFrom(end_time_proto)
 
     return in_msg
+
+
+# Remove when https://github.com/getsentry/eap-planning/issues/206 is merged
+def transform_binary_formula_to_expression(
+    column: Column.BinaryFormula,
+) -> Expression.BinaryFormula:
+    return Expression.BinaryFormula(
```

Comment:
```suggestion
def transform_binary_formula_to_expression(
    column: Column.BinaryFormula,
) -> Expression.BinaryFormula:
    """TODO: Remove when https://github.com/getsentry/eap-planning/issues/206 is merged, since we can use formulas in both APIs at that point"""
    return Expression.BinaryFormula(
```

Reply:
Done!